sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosResponseValidator.java

```diff
@@ -210,7 +210,8 @@ public void validate(CosmosAsyncStoredProcedureResponse resourceResponse) {
                 @Override
                 public void validate(T resourceResponse) {
                     assertThat(resourceResponse.getProperties()).isNotNull();
-                    assertThat(resourceResponse.getProperties().getETag()).isNotNull();
+                    //TODO: Uncomment this after checking getETag()
+//                    assertThat(resourceResponse.getETag()).isNotNull();
```

Comment:
what happened to this guy?

Reply:
Left over from a trial of removing CosmosItemProperties. Got it back.